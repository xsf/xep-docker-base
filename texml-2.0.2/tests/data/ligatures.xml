<TeXML>z &lt;&lt;x&gt;&gt; - -- --- ----- z ` `` ``` ' '' '''' !!!` !` ???` ?` z
	<TeXML ligatures="1">z &lt;&lt;x&gt;&gt; - -- --- ----- z ` `` ``` ' '' '''' !!!` !` ???` ?` z
		<TeXML>z &lt;&lt;x&gt;&gt; - -- --- ----- z ` `` ``` ' '' '''' !!!` !` ???` ?` z
			<TeXML ligatures="0">z &lt;&lt;x&gt;&gt; - -- --- ----- z ` `` ``` ' '' '''' !!!` !` ???` ?` z
				<TeXML>z &lt;&lt;x&gt;&gt; - -- --- ----- z ` `` ``` ' '' '''' !!!` !` ???` ?` z</TeXML>
			</TeXML>
</TeXML>
	</TeXML>
</TeXML>
