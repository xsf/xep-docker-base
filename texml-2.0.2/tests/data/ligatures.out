z \textless{}\textless{}x\textgreater{}\textgreater{} - -{}- -{}-{}- -{}-{}-{}-{}- z ` `{}` `{}`{}` ' '{}' '{}'{}'{}' !!!{}` !{}` ???{}` ?{}` z
z \textless{}\textless{}x\textgreater{}\textgreater{} - -- --- ----- z ` `` ``` ' '' '''' !!!` !` ???` ?` z
z \textless{}\textless{}x\textgreater{}\textgreater{} - -- --- ----- z ` `` ``` ' '' '''' !!!` !` ???` ?` z
z \textless{}\textless{}x\textgreater{}\textgreater{} - -{}- -{}-{}- -{}-{}-{}-{}- z ` `{}` `{}`{}` ' '{}' '{}'{}'{}' !!!{}` !{}` ???{}` ?{}` z
z \textless{}\textless{}x\textgreater{}\textgreater{} - -{}- -{}-{}- -{}-{}-{}-{}- z ` `{}` `{}`{}` ' '{}' '{}'{}'{}' !!!{}` !{}` ???{}` ?{}` z
